roles/generate_discovery_iso_mce/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: Get CA Secret and save to ca-trust path
+  shell: "oc get secret router-ca -o json -n openshift-ingress-operator |jq -r \".data.\\\"tls.crt\\\"\"|base64 -d > /etc/pki/ca-trust/source/anchors/nokianom.crt "
+  register: cert
+
+- name: Update CA Trust
+  command: update-ca-trust
+
+- name: Get ISO Download Link
+  shell: oc get infraenv -A -o json|jq -r '.items[0].status.isoDownloadURL'
+  register: iso_url
+
+- set_fact:
+      iso_url_str: "{{ iso_url.stdout | replace ('minimal' , 'full') }}"
+
+- name: Download ISO
+  get_url:
+    url: "{{iso_url_str}}"
+    dest: /opt/http_store/data/discovery/discovery_mce.iso

```

Comment:
Construct the save path out of these vars that way if the http_store is not the crucible one the use can still modify the location.
```
DOWNLOAD_DEST_FILE: "{{ day2_discovery_iso_name }}"
DOWNLOAD_DEST_PATH: "{{ iso_download_dest_path | default('/opt/http_store/data') }}"
```

Reply:
done